app/views/spree/admin/translations/_form_fields.html.erb

```diff
@@ -13,14 +13,14 @@
                 <% if locale.to_s.include?('-') %>
                   <img src="https://www.localeapp.com/assets/flags/regions/<%= locale.to_s.split('-').last %>.png">
                 <% else %>
-                  <img src="https://www.localeapp.com/assets/flags/languages/<%= locale %>.png">
+                  <img src="https://l10n.xwiki.org/xwiki/bin/download/L10N/Flags/<%= locale %>.png">
```

Comment:
I think we should remove the flags.
Flags symbolizes countries, not languages.
In any case, the images should never be loaded from an external source.